static/src/components/CardHist.js

```diff
@@ -0,0 +1,49 @@
+import React, { PropTypes } from 'react';
+import { Card, CardText, CardHeader, CardMedia, CardTitle, GridList } from 'material-ui';
+
+function CardHist(props) {
+    console.log('==>', props.historyData);
```

Comment:
We should remove console.log errors before merging to master

Reply:
One day I will remember